app/views/govuk_publishing_components/components/_layout_for_admin.html.erb

```diff
@@ -9,15 +9,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <%= csrf_meta_tags %>
     <%= favicon_link_tag "govuk_publishing_components/favicon-#{environment}.png" %>
-    <%= stylesheet_link_tag "govuk_publishing_components/admin_styles" %>
+    <%= stylesheet_link_tag "application" %>
```

Comment:
Rather than using this by convention we could use the variables from the config: https://github.com/alphagov/govuk_publishing_components/blob/d8a98757dcacccdf95d9de48feac22d0b497cf5f/app/views/layouts/govuk_publishing_components/application.html.erb#L20-L24

Which will make it less painful if someone doesn't use the convention.

Reply:
We can stick to the config as @kevindew suggested.

Reply:
I'd like to go with the convention of using `application.js`. It's one of the things we track in [the consistency spreadsheet](https://docs.google.com/spreadsheets/d/1J-0IPyr2ycPuw1IWOIoX0o9Tk9nF35D0DJdsj_ylLQE/edit#gid=0), so apps should not diverge from this intentionally.